grpc-sys/build.rs

```diff
@@ -72,6 +72,9 @@ fn build_grpc(cc: &mut Build, library: &str) {
         if cfg!(target_os = "macos") {
             config.cxxflag("-stdlib=libc++");
         }
+        if cfg!(feature = "openssl") {
+            config.define("gRPC_SSL_PROVIDER", "package");
```

Comment:
as mentioned in my PR comment I think you want to define EMBED_OPENSSL=false instead.